tools/build/lib/byond.js

```diff
@@ -60,9 +60,10 @@ const getDmPath = async () => {
 
 /**
  * @param {string} dmeFile
+ * @param {boolean} error
  * @param {{ defines?: string[] }} options
  */
-export const DreamMaker = async (dmeFile, options = {}) => {
+export const DreamMaker = async (dmeFile, error = false, options = {}) => {
```

Comment:
You must put this in the `options` argument, e.g.
```ts
{
  defines?: string[];
  warningsAsErrors?: boolean;
}
```

Reply:
Done. Please check if I've written jsdoc correctly, it's my first time using it.